build/nuke/docker/alpine.dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM mcr.microsoft.com/dotnet/sdk:6.0-alpine3.16
+
+RUN apk update \
+    && apk upgrade \
+    && apk add --no-cache --update \
+        clang \
+        cmake \
+        make \
+        bash \
+        alpine-sdk \
+        protobuf \
+        protobuf-dev \
+        grpc
+
+ENV IsAlpine=true
+ENV PROTOBUF_PROTOC=/usr/bin/protoc
+ENV gRPC_PluginFullPath=/usr/bin/grpc_csharp_plugin
+
+# Install .NET Core older sdks using install script
+RUN curl -sSL https://dot.net/v1/dotnet-install.sh --output dotnet-install.sh \
```

Comment:
Here we should at least verify checksum of downloaded file.

Reply:
Do we have a way to automatically get the appropriate checksum? My assumption is that it will change with the monthly security updates.

Reply:
On our side, we are updating this value manually, when the scripts starts failing.
I am not telling that we have to do this in this PI, but we should consider how we will update this kind of dependencies (I can imagine that we create action to auto update value. The PR could contains differences between old and new script value).

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/issues/1137